omnisend/includes/SDK/V1/class-getcontactresponse.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Omnisend Client
+ *
+ * @package OmnisendClient
+ */
+
+namespace Omnisend\SDK\V1;
+
+use WP_Error;
+
+defined( 'ABSPATH' ) || die( 'no direct access' );
+
+class GetContactResponse {
+
+	private array $contact_data;
```

Comment:
We should not expose raw array data, because for SDK user without deeper Omnisend API knowledge will not be clear what structure is behind this array.

Also you mapped only few properties required I guess for your usecase. If new contact property will be required, for example phone it will be pretty difficult to get it. 

Please do following changes:
* leave only two properties in GetContactResponse class (contact and wp_error)
* contact should be not array, but Contact class (need to map from array)
* add getters (or additional methods if needed with prefixt `Has..` or `Is..`) to Contact class as Contact properties are private